recipes/ixwebsocket/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "ixwebsocket" %}
+{% set version = "11.4.5" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/machinezone/IXWebSocket/archive/v{{ version }}.tar.gz
+  sha256: c5fc225edec32bf7d583e55347ef2c9c4940d005c13ef5e848354a85602f5fd6
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+  host:
+    - zlib
+    - openssl
+  run:
+
+outputs:
+  - name: libixwebsocket
+    build:
+      run_exports:
+        - {{ pin_subpackage('libixwebsocket') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
+      host:
+        - zlib
+        - openssl
+      run:
+    files:
+      - lib/libixwebsocket.so.*  # [linux]
+      - lib/libixwebsocket.*.dylib  # [osx]
+    test:
+      commands:
+        - test ! -f ${PREFIX}/lib/libixwebsocket${SHLIB_EXT}  # [unix]
+
+  - name: ixwebsocket
+    build:
+      run_exports:
+        - {{ pin_subpackage('libixwebsocket') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
+      host:
+        - {{ pin_subpackage('libixwebsocket', exact=True) }}
+      run:
+        - {{ pin_subpackage('libixwebsocket', exact=True) }}
+    files:
+      - lib/libixwebsocket.so  # [linux]
+      - lib/libixwebsocket.dylib  # [osx]
+      - include/ixwebsocket  # [unix]
```

Comment:
Should these be in the lib output?

Reply:
In principle no. They will be used during the linking phase of a build against the lib, but then the dependent package will have a requirement on the versioned version of libixwebsocket. Cf https://github.com/conda-forge/zlib-feedstock/blob/main/recipe/meta.yaml for a similar situation.